CalibPPS/TimingCalibration/plugins/PPSDiamondSampicTimingCalibrationPCLWorker.cc

```diff
@@ -62,22 +62,35 @@ class PPSDiamondSampicTimingCalibrationPCLWorker
                   edm::EventSetup const&,
                   Histograms_PPSDiamondSampicTimingCalibrationPCLWorker const&) const override;
 
+  template <typename T>
+  bool searchForProduct(edm::Event const& iEvent,
+                        const std::vector<edm::EDGetTokenT<T>>& tokens,
+                        const std::vector<edm::InputTag>& tags,
+                        edm::Handle<T>& handle) const;
+
   // ------------ member data ------------
-  edm::EDGetTokenT<edm::DetSetVector<TotemTimingDigi>> totemTimingDigiToken_;
-  edm::EDGetTokenT<edm::DetSetVector<TotemTimingRecHit>> totemTimingRecHitToken_;
+  std::vector<edm::InputTag> digiTags_;
+  std::vector<edm::InputTag> RecHitTags_;
+
+  std::vector<edm::EDGetTokenT<edm::DetSetVector<TotemTimingDigi>>> totemTimingDigiTokens_;
+  std::vector<edm::EDGetTokenT<edm::DetSetVector<TotemTimingRecHit>>> totemTimingRecHitTokens_;
```

Comment:
similarly can these be `const`-s?

Reply:
Ok, to sum up, I added const to every member variable in these workers that can be const.

Reply:
Thanks @MatiXOfficial !